src/components/ExportPanel.vue

```diff
@@ -132,8 +132,8 @@ export default {
         timer: this.$store.state.session.timer,
         started: this.$store.state.session.started,
         ended: this.$store.state.session.ended,
-        reportLogo: this.config.reportLogo,
-        logoPath: this.config.logoPath,
+        reportLogo: logo && logo.enabled,
+        logoPath: logo && logo.path && logo.path.path,
```

Comment:
@dockabernathy01 - what is `logo.path.path`? I don’t see the nested `path` variables anywhere else

Reply:
@dacoaster , I have stored the path like this : 
    "path": {
      "path": "E:\\Mine\\Electron\\Yattie-Session\\logo.png",
      "name": "logo.png",
      "size": 1610
    }

Reply:
![logo](https://github.com/yatt-ai/yattie/assets/146516559/8292905e-834e-4179-b3b4-b1821976df1c)